src/reducers/publicOrderReducer.js

```diff
@@ -0,0 +1,9 @@
+export default function(state = 'Most Recent', action) {
+  switch (action.type) {
+    case 'SET_PUBLIC_ORDER':
+      state = action.publicOrder;
+      return state;
```

Comment:
@thevaleriemack `return action.publicOrder` :+1: then this is clearly a pure function, as reducers ought to be (and is!)